receiver/prometheusoperatorreceiver/README.md

```diff
@@ -0,0 +1,80 @@
+# Simple Prometheus Receiver
+
+The `prometheus_operator` receiver is a wrapper around the [prometheus
+receiver](../prometheusreceiver).
+This receiver allows using [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) 
+CRDs like `ServiceMonitor` and `PodMonitor` to configure metric collection.
+
+Supported pipeline types: metrics
+
+> :construction: This receiver is in **development**, and is therefore not usable at the moment.
+
+## When to use this receiver?
+Choose this receiver, if you use [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/example/user-guides/getting-started/example-app-service-monitor.yaml) 
+or [PodMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/example/user-guides/getting-started/example-app-pod-monitor.yaml) 
+custom resources in your Kubernetes cluster. These can be provided by applications (e.g. helm charts) or manually 
+deployed by users. 
+
+In every other case other Prometheus receivers should be used. 
+Below you can find a short description of the available options.
+
+### Prometheus scrape annotations
```

Comment:
I don't think you even need to use the receivercreator in this case.  You can just use the __meta_kubernetes_pod_annotation_prometheus_io_scrape label to filter pods (use the equivalent for endpoints) directly in the prometheusreceiver.

Reply:
Should work as you describe it. I agree that the `prometheusreceiver` is in that case preferable.    
I will adapt here the section too.